addons/l10n_ro_edi/models/account_edi_xml_ubl_ciusro.py

```diff
@@ -89,6 +115,19 @@ def _export_invoice_constraints(self, invoice, vals):
         # EXTENDS 'account_edi_ubl_cii'
         constraints = super()._export_invoice_constraints(invoice, vals)
 
+        # Default VAT is only allowed for the receiver (customer), not the provider (supplier)
+        if (
+            (
+                not vals['supplier'].commercial_partner_id.vat
+                or len(vals['supplier'].commercial_partner_id.vat) == 1
+            )
+                and not vals['supplier'].commercial_partner_id.company_registry
+        ):
```

Comment:
```suggestion
        supplier = vals['supplier'].commercial_partner_id
        if (
            (
                not supplier.vat
                or len(supplier.vat) == 1
            )
                and not supplier.company_registry
        ):
```
More pleasing to the eye, no ?